extensions/narayana-jta/runtime/src/main/java/io/quarkus/narayana/jta/runtime/ObjectStoreType.java

```diff
@@ -0,0 +1,6 @@
+package io.quarkus.narayana.jta.runtime;
+
+public enum ObjectStoreType {
+    FileSystem,
```

Comment:
Shouldn't this be `FILE_SYSTEM`? That's the convention in most Java codestyles I've seen, at least.

Note Quarkus will automatically accept `file-system` in configuration files even if your enum is named `FILE_SYSTEM`.

Reply:
Done.